build.gradle

```diff
@@ -159,7 +159,7 @@ def toPackageClasspath = files(
 sourceSets {
     main {
         java {
-            srcDir "src"
+            srcDirs = ["src", "tests"]
```

Comment:
Make sense. I've changed to only include the test framework code in inference jar.